libs/vre/shared/app-resource-page/src/lib/resource-representation.component.ts

```diff
@@ -15,25 +15,23 @@ import { filter, map } from 'rxjs/operators';
 
 @Component({
   selector: 'app-resource-representation',
-  template: ` <div class="representation-container center" [ngSwitch]="representationToDisplay.fileValue.type">
-    <app-still-image
-      #stillImage
-      class="dsp-representation stillimage"
-      [attachedProject]="attachedProject$ | async"
-      *ngSwitchCase="representationConstants.stillImage"
-      [resource]="resource.res" />
-    <app-still-image
-      #stillImage
-      class="dsp-representation stillimage"
-      *ngSwitchCase="representationConstants.externalStillImage"
-      [attachedProject]="attachedProject$ | async"
-      [resource]="resource.res" />
-
+  template: ` <div class="representation-container center" [ngSwitch]="true">
```

Comment:
IMO leaving as it was before `[ngSwitch]="representationToDisplay.fileValue.type"` is better, because you don't need to use `===` operator, less code and easier to read. So at the end I think this is what @derschnee68 mean in his comment few lines below, right?